build-logic/java-api-extractor/src/main/java/org/gradle/internal/tools/api/ApiMemberWriter.java

```diff
@@ -40,7 +40,7 @@ public interface ApiMemberWriter {
 
     void writeClass(ClassMember classMember, Set<MethodMember> methods, Set<FieldMember> fields, Set<InnerClassMember> innerClasses);
 
-    void writeMethod(MethodMember method);
+    void writeMethod(/* Nullable */ InnerClassMember declaringInnerClass, MethodMember method);
```

Comment:
💭 `/* Nullable */` is something IntelliJ actually tracks in place of `@Nullable`.https://github.com/gradle/gradle/pull/31417

We don't have access to `@Nullable` here, so...